_component_design/alert-bar.md

```diff
@@ -14,77 +14,110 @@ last-modified: 2017-08-17
+to the status of the item. It can be scoped across the entirety of
+the content area (page level scoping) or scoped to extend across a
+singular section of the content area (element level scoping).
 
-Ideally alert bars are used to communicate things like “unread messages”, “page
-errors”, or confirmations for an action that happened off canvas or in a
-previous screen like a modal or form.
+{% figure [caption:"Alert bar hero"] [class:"image bg-light border"] %}
+ ![]({{site.url}}/assets/images/components/content-areas/alerts/alert-bar-hero.svg){:width="80%"}
+{% endfigure %}
 
-Alert bars can also provide some basic quick instruction for a page to better
-assist the user with understanding the task or feature they can/need to
-perform.
+## When to use alerts
 
-<div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Alert bar at a page level"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/alert-bar-page.svg)
-{% endfigure %}
-{% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Alert bar at a region level"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/alert-bar-region.svg)
-{% endfigure %}
-{% endcolumn %}
-</div>
+Notification bars are used to provide status information about a page or
+element to a user.
+
+* Notification bars are best used for information that will remain static,
+  barring user intervention. For example, an informative message surrounding a new
+  feature being made available to a user. For information that is more
+  ephemeral and fleeting, we recommend using another notification
+  type.
+
+## Best practices for alerts
+
+We recommend following these best practices:
+
+* Ensure a notification bar is placed across the entirety of the
+  referenced element.
+* Ensure notification bars have implicit priority based on the alert type from most
+  to least severity level:
+  * Error > Warning > Success > Informational
+* Ensure notification bars are used sparingly and limited to one per page.
+  * In the event a higher priority alert bar needs to be shown on a page that
+    already contains a  notification bar. The higher level alert is placed
+    *above* the lower priority bar.
+
+# Specifications
+
+## Default layout
 
-# Anatomy
+Alert bars rest at the top of the content section of a page layout in most
+situations, extending across the entire section.
 
-<div class="hxRow">
-{% column left:"hxCol-4" %}
-The alert bar consists of 3 major parts ALWAYS:
+* Matches the margin of the content area to which it relates.
 
-- Bar: enclosing shape to frame and present the alert
-- Icon: which provides context to the classification of the alert type
-- Message: provides details and/or instructions to give the user context
+{% figure [caption:"When you need to describe an image in greater detail, use this
+ block of text so that you can keep the left area clutter-free. 3 lines or less."] [class:"image bg-light border"] %}
```

Comment:
nix the placeholder caption.  Sorry bout these -- we'll make sure the designers remove them when they don't need them.